packages/core/upload/admin/src/components/TableList/CellContent.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { getFileExtension } from '@strapi/helper-plugin';
+import { Typography } from '@strapi/design-system/Typography';
+
+import { PreviewCell } from './PreviewCell';
+import { formatBytes } from '../../utils';
+
+export const CellContent = ({
+  alternativeText,
+  content,
+  cellType,
+  elementType,
+  mime,
+  fileExtension,
+  thumbnailURL,
+  url,
+}) => {
+  const { formatDate } = useIntl();
+  switch (cellType) {
+    case 'image':
+      return (
+        <PreviewCell
+          alternativeText={alternativeText}
+          fileExtension={fileExtension}
+          mime={mime}
+          type={elementType}
+          thumbnailURL={thumbnailURL}
+          url={url}
+        />
+      );
+    case 'date':
+      return <Typography>{formatDate(content)}</Typography>;
+
+    case 'size':
+      if (elementType === 'folder') return <Typography>-</Typography>;
+
+      return <Typography>{formatBytes(content)}</Typography>;
```

Comment:
Are we displaying only the date or datetime here? I'm wondering if it would make sense to unify this with the behavior in the CM: https://github.com/strapi/strapi/blob/45831f30f5993709fa0fe06222578603d0cc4eb9/packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/CellValue.js#L10-L16

Reply:
[fixed](https://github.com/strapi/strapi/pull/15085/commits/dc055b43710f3606b5f450f318f99638d50ae00e)

Reply:
Sounds good to me, thanks!